lib/extensions/ar_application_name.rb

```diff
@@ -4,10 +4,6 @@ module ArApplicationName
   # connections in the pool.  We do this because reconnect! on an instance of the
   # AR adapter created prior to our change to PGAPPNAME will have old connection
   # options, which will reset our application_name.
-  #
-  # Because we fork workers from the server, if we don't disconnect the pool,
-  # any call to reconnect! on a connection will cause the worker's connection
-  # to have the server's application_name.
```

Comment:
Note, this is still needed in general because the first connection to the db will happen before we can set the application name, so it defaults to the rails application name. The comment is no longer relevant though.